core-api/api_container_service.proto

```diff
@@ -24,6 +24,9 @@ service ApiContainerService {
 
   // Executes the given command inside a running container
   rpc ExecCommand(ExecCommandArgs) returns (ExecCommandResponse) {};
+
+  //Checks service availability
+  rpc CheckAvailability(CheckAvailabilityArgs) returns (google.protobuf.Empty) {};
```

Comment:
Can we call this `WaitForEndpointAvailability`, to more accurately reflect its purpose?

Reply:
Yes! I'll do it